jac-uk/admin
# update application to use new editablefields

## What's included?

Updated the application view to use new versions of the `EditableField` component.
Citizenship†, Reasonable adjustments†, and Reasonable adjustments details* have can now be updated by admins.
† new dropdown selection type
* textfield
 
## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- View any application to any exercise.
- under personal details you should see that Citizenship, Reasonable adjustments, and Reasonable adjustments details can now be updated.
- Make changes
- Save
- Reload page, changes persist

[bonus round] - Check info changes on apply portal too

## Risk - how likely is this to impact other areas?
🟢 Low risk - this is a self-contained piece of work

## Additional context

![image](https://user-images.githubusercontent.com/44227249/119023936-6ffebe00-b99a-11eb-9e7d-c6706a4562f2.png)
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
